### ISSUE TITLE
Missing Properties

### ISSUE THREAD

**Issue by ollyhensby** [buildingSMART/bSDD#32]
**Describe the bug**
Missing properties when using the "Get Properties" function.

![image](https://user-images.githubusercontent.com/79875276/165741799-c12df6a2-5649-48bd-b862-3497e5f92477.png)

**To Reproduce**
For example, I want to obtain properties listed within a property set. Let's look at _Pset_BuildingUse_: 
http://ifc43-docs.standards.buildingsmart.org/IFC/RELEASE/IFC4x3/HTML/lexical/Pset_BuildingUse.htm

Now if I search for these properties in the API, some of them return results. However, _TenureModesAvailableNow_ does not. Other properties are missing too.

It seems from a little experimenting that only IFC properties with the property type _IfcPropertySingleValue_ can be called?

Maybe I am just looking at the incorrect documentation that is associated with the API?

http://identifier.buildingsmart.org/uri/buildingsmart/ifc-4.3/prop/TenureModesAvailableNow

![image](https://user-images.githubusercontent.com/79875276/165742232-9a16e775-601d-42e6-b158-2e2a0bd33026.png)

**Expected behaviour**
I expect that the properties that exist in the documentation for ifc 4.3 should be able to be called from the API property function.

**Screenshots**
See above

**Desktop (please complete the following information):**
 - OS: [Windows 10]
 - Browser [Chrome]

Any help with this will be much appreciated! Thank you

**Comment by aothms** [comment]
Good point. Currently we indeed include only SingleValue and EnumeratedValue (https://github.com/buildingSMART/IFC4.3.x-development/blob/master/code/to_bsdd.py#L202-L214)

We have talked in the past about BoundedValue and ListValue but I forgot the conclusions. @Baars-IT @berlotti @grandfr perhaps anyone remembers?

**Comment by ollyhensby** [comment]
@aothms Okay, thanks for clarifying about the property types. Any update on adding the other property types?

**Comment by paoloborin** [comment]
Following! any update? Thanks

**Comment by atomczak** [comment]
yes, we are now in the process of adding all missing IFC types of properties to bSDD, also Lists and Tables. While it might take a few weeks, the solution is on its way. Sorry to keep you waiting :)
